examples/bert/prepare_data.py

```diff
@@ -35,25 +35,67 @@
     "The task to run experiment on. One of "
     "{'COLA', 'MNLI', 'MRPC', 'XNLI', 'SST'}.")
 flags.DEFINE_string(
-    "vocab_file", 'bert_pretrained_models/uncased_L-12_H-768_A-12/vocab.txt',
-    "The one-wordpiece-per-line vocabary file directory.")
+    "pretrained_model_name", 'bert-base-uncased',
+    "The name of a pre-trained model to load selected in the list of: "
+    "`bert-base-uncased`, `bert-large-uncased`, "
+    "`bert-base-cased`, `bert-large-cased`, "
+    "`bert-base-multilingual-uncased`, `bert-base-multilingual-cased`, "
+    "and `bert-base-chinese`.")
 flags.DEFINE_integer(
     "max_seq_length", 128,
-    "The maxium length of sequence, longer sequence will be trimmed.")
+    "The maximum length of sequence, longer sequence will be trimmed.")
 flags.DEFINE_string(
     "tfrecord_output_dir", None,
     "The output directory where the TFRecord files will be generated. "
     "By default it will be set to 'data/{task}'. E.g.: if "
     "task is 'MRPC', it will be set as 'data/MRPC'")
-flags.DEFINE_bool(
-    "do_lower_case", True,
-    "Whether to lower case the input text. Should be True for uncased "
-    "models and False for cased models.")
 
 tf.logging.set_verbosity(tf.logging.INFO)
 
 
-def prepare_data():
+def modify_config_data(max_seq_length, num_train_data, num_classes):
```

Comment:
If this is not for users to use (or care about), rename it to `_modify_config_data` to be a private function